File: Architecture.Core/Entities/SectionLink.cs
using Architecture.Core.Entities.Core;
using Architecture.Core.Entities.LU;

namespace Architecture.Core.Entities
{
    public class SectionLink : Auditable
    {
        public int SectionLinkId { get; set; }
        public string Title { get; set; }
        public string ActionLink { get; set; }
        public int? SectionNameId { get; set; }
        public virtual SectionName SectionName { get; set; }
        public string Remarks { get; set; } 

    }
}

File: Architecture.Web/ClientApp/src/app/Shared/Entity/Users/SectionLink.ts
import { Data } from '@angular/router';
import { Auditable } from '../Common/auditable';

export class SectionLink extends Auditable {
    public SectionLinkId: number;    
    public title: string;
    public actionLink: string;   
    public sectionName: string;
    public sectionNameId: number;
    public remarks: any;    
}


